config/routes.rb

```diff
@@ -8,5 +11,5 @@
       patch :withdrawal
     end
   end
-  resources :tweets
+  resources :tweets, :comments
```

Comment:
resourcesはonlyオプションを定義することを考慮して別々に定義したほうがよいです。

```suggestion
  resources :tweets
  resources :comments
```